realsense2_camera/CMakeLists.txt

```diff
@@ -83,17 +83,11 @@ catkin_package(
     )
 
 add_library(${PROJECT_NAME}
-    include/constants.h
-    include/realsense_node_factory.h
-    include/base_realsense_node.h
-    include/rs415_node.h
-    include/rs435_node.h
-    include/sr300_node.h
-    src/realsense_node_factory.cpp
-    src/base_realsense_node.cpp
-    src/rs415_node.cpp
-    src/rs435_node.cpp
-    src/sr300_node.cpp
+    src/realsense_nodelet.cpp
+    src/realsense_node.cpp
+    src/rs415_param_manager.cpp
+    src/rs435_param_manager.cpp
```

Comment:
Correct me if I'm being blind here, but I don't see either of these files (`src/rs415_param_manager.cpp` and `src/rs435_param_manager.cpp`). It looks like the previous files they're replacing (`src/rs415_node.cpp` and `src/rs435_node.cpp`) just got deleted.